Core.Tests/StateManagerTests.cs

```diff
@@ -1,10 +1,54 @@
-using Xunit;
+using FluentAssertions;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+using System.Collections.Generic;
```

Comment:
Codacy found an issue: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=6081671)